configs/prometheus.example.yaml

```diff
@@ -0,0 +1,8 @@
+---
+global:
+  scrape_interval: 3s
+
+scrape_configs:
+  - job_name: idmsvc_backend
+    static_configs:
+      - targets: ['host.containers.internal:9000']
```

Comment:
The current configuration does not let me pull the metrics from the service running locally.

I need to check the above in different systems:
- `localhost:9000` this is necessary on Linux when the container is launched in the host network, to pull the metrics from the service running locally.
- `host.containers.internal:9000` probably is necessary when running the container with podman, but this does not let me pull the metrics from the service running in my linux box.
- `host.docker.internal:9000` similar for docker (based on the entry here: https://stackoverflow.com/questions/56909896/prometheus-in-docker-container-cannot-scrape-target-on-host).

Probably some finetunes to the makefile is needed.

Reply:
Kinda like the idea to include more and it will scrape the ones that works. When testing this, it looks like Podman works with both `host.containers.internal:9000`, `host.docker.internal:9000`.  Thus I'm thinking of dropping `host.containers.internal:9000` to scrape only one endpoint while maintaining compatibility with Docker.

I don't have Docker ready to test other stuff.

Reply:
Changed to `host.docker.internal:9000`